lib/access.js

```diff
@@ -61,17 +66,22 @@ function access (args, cb) {
 
   function invokeCmd (err, uri, auth, base) {
     if (err) { return cb(err) }
-    params.auth = auth
-    try {
-      return npm.registry.access(cmd, uri, params, function (err, data) {
-        if (!err && data) {
-          output(JSON.stringify(data, undefined, 2))
-        }
-        cb(err, data)
+    return Bluebird.try(() => {
+      params.auth = auth
+      try {
```

Comment:
This is super weird and uncomfortable and actually what the API for this function is... =/